ontology/yaml/resources/HVAC/entity_types/DMP.yaml

```diff
@@ -99,6 +99,14 @@ DMP_ZSPC_EFM:
   - ZSPC
   - EFM
 
+DMP_RAIDC:
```

Comment:
Is this return air damper part of a larger system are there more points available? When possible we want to model the entire system instead of one part. For example, if this is a return air damper for a air handling unit, then we want to model the entire air handling unit with an abstract that includes the return air damper points.

Reply:
Alright, I'll submit a pull request for the FCU canonical type. Thank you; I'm closing this pull request now.